conf/layer.conf

```diff
@@ -17,6 +17,7 @@ BBFILES_DYNAMIC += " \
     openembedded-layer:${LAYERDIR}/dynamic-layers/openembedded-layer/*/*/*.bbappend \
     networking-layer:${LAYERDIR}/dynamic-layers/networking-layer/*/*/*.bb \
     networking-layer:${LAYERDIR}/dynamic-layers/networking-layer/*/*/*.bbappend \
+    meta-python2:${LAYERDIR}/dynamic-layers/meta-python2/*/*/*.bb \
```

Comment:
can we avoid this please ? py2 is dead and I prefer that we can stay away from it.

Reply:
I will have to add a patch against llvm-project-source so libclc would look for `python3` instead of `python`.
Then it can be moved back to `recipes-devtools/clang`.

Reply:
yes such a patch is good and we can upstream it too if not we can maintain it in meta-clang

Reply:
I added the patch, still llvm is compiling on my side but I pushed the changes.
I also added a patch from https://bugs.llvm.org/show_bug.cgi?id=48921 which fixes broken radeonsi shader compilation in some cases, as reported in https://gitlab.freedesktop.org/mesa/mesa/-/issues/4107